erpnext/stock/doctype/material_request_item/material_request_item.json

```
@@ -444,16 +445,23 @@
   {
    "fieldname": "qty_info_col_break",
    "fieldtype": "Column Break"
+  },
+  {
+   "fieldname": "job_card_item",
+   "fieldtype": "Data",
+   "hidden": 1,
```

```suggestion
   "hidden": 1,
   "no_copy": 1,
   "print_hide": 1,
```

To avoid accidentally copying references over to duplicate entry